### DISCUSSION TITLE
How do I record audio in .NET MAUI?

### DISCUSSION THREAD

**Discussion by r2d2Proton** [dotnet/maui#16133] [Q&A]
I can Capture an Image easily. And Capture Video easily. It seems something as fundamental as audio was skipped. I did find this:

https://www.syncfusion.com/blogs/post/building-an-audio-recorder-and-player-app-in-net-maui.aspx

**Comment by Xyncgas** [comment]
You can always create a blazor web view with 0 width and height in your app and do what you need to do using browser's way

**Comment by davidortinau** [comment]
That looks like a great, thorough article from Syncfusion. 

If you just want speech recognition, this article may be easier:

https://devblogs.microsoft.com/dotnet/speech-recognition-in-dotnet-maui-with-community-toolkit/

I see there are also a few plugins on NuGet that provide recording features as well. 

https://github.com/math3ussdl/Plugin.Maui.AudioRecorder
https://github.com/hjam40/Audio.MAUI
